data/VISHAL3545.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Vishal Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Data Engineer turned *Content Creator*",
+  "avatar": "https://github.com/VISHAL3545.png",
```

Comment:
In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that. If you need any help feel free to ask.

Updated docs: https://linkfree.eddiehub.io/docs/quickstart